src/policy/semantic.rs

```diff
@@ -415,9 +415,11 @@ impl<Pk: MiniscriptKey> Policy<Pk> {
 
                 let n = subs.len() - unsatisfied_count - trivial_count; // remove all true/false
                 let m = k.checked_sub(trivial_count).map_or(0, |x| x); // satisfy all trivial
-                                                                       // m == n denotes `and` and m == 1 denotes `or`
+
+                // m == n denotes `and` and m == 1 denotes `or`
                 let is_and = m == n;
                 let is_or = m == 1;
```

Comment:
In 939695dfcb6508e0062a50b80e5e68dcfa5b0e4b:

We can just delete this comment I think. I can't imagine what value it provides beyond the code itself.